docs/install.rst

```diff
@@ -95,38 +143,41 @@ To test a single app, specify the app by name in the command above. e.g.::
     copying ``docker/envfiles/demo.env`` to ``.env``, and running tests again.
 
 Make it run
------------
-
-To make the server run, make sure you are inside a virtualenv, and then
-run the server::
-
-    $ ./manage.py runserver
-
-If you are not inside a virtualenv, you can activate it by doing::
+===========
 
-    $ source venv/bin/activate
+Docker
+------
 
-If you get the error "NoneType is not iterable", you didn't check out the latest product-details. See the above section for that.
+You can simply run the ``make run`` script mentioned above, or use docker-compose directly::
 
-Front-end assets
-~~~~~~~~~~~~~~~~
+    $ docker-compose up app assets
 
-Next, in a new terminal tab run gulp to watch for local file changes::
+Local
+-----
 
-    $ gulp
+To make the server run, make sure your virtualenv is activated, and then
+run the server::
 
-This will automatically copy over CSS, JavaScript and image files to the /static directory as and when they change, which is needed for Django Pipeline to serve the assets as pages are requested.
+    $ npm start
 
-If you have problems with gulp, or you for some reason don't want to use it you can set::
+If you are not inside a virtualenv, you can activate it by doing::
 
-    PIPELINE_COLLECTOR_ENABLED=True
+    $ source venv/bin/activate
 
-in your ``.env`` file or otherwise set it in your environment and it will collect media for you as you make changes. The reason that this is not the preferred method is that it is much slower than using gulp.
+Browsersync
```

Comment:
@bensternthal I added this section about browsersync and how it works and potential issues. Do you think this is sufficient?